### PR TITLE
fix: Minor changes to specifications 0013-ACCT-accounts.md

### PR DESCRIPTION
**Pull request by MuthuVega**
Minor changes to specs.

### CODE REVIEW

File: protocol/0013-ACCT-accounts.md [hunk 1]
```diff
@@ -24,16 +24,6 @@ Note that a party can also associate the governance / staking asset via the [Veg
 
 One key difference with staking accounts is that the collateral is not held in an asset bridge, but in the [staking bridge](./0071-STAK-erc20_governance_token_staking.md). The balance is changed by events on Ethereum, rather than actions taken on the Vega chain.
 
-## Liquidity Provider bond accounts
-- [ ] A bond account holds collateral to maintain collateral for [Liquidity Providers](./0044-LIQM-lp_mechanics.md). (<a name="0013-ACCT-017" href="#0013-ACCT-017">0013-ACCT-017</a>)
```
Comment (davidsiska-vega) [inline protocol/0013-ACCT-accounts.md:28]: Why is this being deleted?

File: protocol/0013-ACCT-accounts.md [hunk 2]
```diff
@@ -79,12 +65,6 @@ When a trader places an order on a market and they do not have a margin account
 * it's the first time a trader has placed an order or;
 * they've previously had a margin account but it was deleted for the reason listed below.
 
-**Deletion:**
```
Comment (davidsiska-vega) [inline protocol/0013-ACCT-accounts.md:82]: Why is this being removed?

File: protocol/0013-ACCT-accounts.md [hunk 3]
```diff
@@ -96,8 +76,6 @@ Every market will have at least one insurance pool account that holds collateral
 
 When a [market launches](./0043-MKTL_market_livecycle.md), an insurance pool account is created for that market for each settlement asset. This account is used by the protocol during the collection of [margin requirements](./0010-MARG-margin_orchestration.md) and the collection of [mark to market settlement](./0003-MTMK-mark_to_market_settlement.md). 
 
-**Deletion:**
```
Comment (davidsiska-vega) [inline protocol/0013-ACCT-accounts.md:99]: Why is this being removed?

File: protocol/0013-ACCT-accounts.md [hunk 4]
```diff
@@ -118,13 +96,13 @@ Note that it *is* possible to have markets in the governance asset, in which cas
   -  [ ] Only one general asset account exists per party per asset. (<a name="0013-ACCT-004" href="#0013-ACCT-004">0013-ACCT-004</a>)
   -  [ ] When a party deposits collateral onto Vega, the asset account will increase in balance by the same amount. (<a name="0013-ACCT-005" href="#0013-ACCT-005">0013-ACCT-005</a>)
   -  [ ] When a party withdraws collateral onto Vega, the asset account for that asset will decrease in balance by the same amount. (<a name="0013-ACCT-006" href="#0013-ACCT-006">0013-ACCT-006</a>) 
+  - [ ] [Fees earned from liquidity provision](./0044-LIQM-lp_mechanics.md#fees) are paid in to this account. (<a name="0013-ACCT-011" href="#0013-ACCT-011">0013-ACCT-011</a>)
```
Comment (davidsiska-vega) [inline protocol/0013-ACCT-accounts.md:106]: I think this is a change you want.

File: protocol/0013-ACCT-accounts.md [hunk 5]
```diff
@@ -118,13 +96,13 @@ Note that it *is* possible to have markets in the governance asset, in which cas
...
 
 ### Party margin accounts
 - [ ] Every party that submits an order on a market will have a margin account for that market created. (<a name="0013-ACCT-007" href="#0013-ACCT-007">0013-ACCT-007</a>)
 - [ ] Each party should only have one margin account per market. (<a name="0013-ACCT-008" href="#0013-ACCT-008">0013-ACCT-008</a>)
 - [ ] Cannot have a non-zero balance on a margin account where there's no position / position size = 0 and no active orders. (<a name="0013-ACCT-009" href="#0013-ACCT-009">0013-ACCT-009</a>)
 - [ ] Cannot transfer into or out of a margin account where there's no position / position size = 0 and no active orders. (<a name="0013-ACCT-010" href="#0013-ACCT-010">0013-ACCT-010</a>)
-- [ ] [Fees earned from liquidity provision](./0044-LIQM-lp_mechanics.md#fees) are paid in to this account. (<a name="0013-ACCT-011" href="#0013-ACCT-011">0013-ACCT-011</a>)
```
Comment (davidsiska-vega) [inline protocol/0013-ACCT-accounts.md:127]: And this is a change you want.

File: protocol/0013-ACCT-accounts.md [hunk 6]
```diff
@@ -133,15 +111,16 @@ Note that it *is* possible to have markets in the governance asset, in which cas
 
 ### Insurance pool accounts
 - [ ] When a market opens for trading, there is an insurance account that is able to be used by that market for every settlement asset of that market. (<a name="0013-ACCT-020" href="#0013-ACCT-020">0013-ACCT-020</a>)
-- [ ] Only transfer requests move money in or out of the insurance account. (<a name="0013-ACCT-021" href="#0013-ACCT-021">0013-ACCT-021</a>)
```
Comment (davidsiska-vega) [inline protocol/0013-ACCT-accounts.md:136]: This change has already been merged - not sure why it's here again.

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
I think something strange happened with `git`. The PR removes / changes a bunch of things it shouldn't in my opinion. Could you please have a look?

File: protocol/0013-ACCT-accounts.md (hunk 1 above)
Comment (MuthuVega) [reply protocol/0013-ACCT-accounts.md:28]: This section was duplicated , hence deleted.

File: protocol/0013-ACCT-accounts.md (hunk 2 above)
Comment (MuthuVega) [reply protocol/0013-ACCT-accounts.md:82]: As discussed, changed label to Creation/Deletion and have left the content intact.

File: protocol/0013-ACCT-accounts.md (hunk 3 above)
Comment (MuthuVega) [reply protocol/0013-ACCT-accounts.md:99]: As discussed, changed label to Creation/Deletion and have left the content intact.

File: protocol/0013-ACCT-accounts.md (hunk 4 above)
Comment (MuthuVega) [reply protocol/0013-ACCT-accounts.md:106]: Yes

File: protocol/0013-ACCT-accounts.md (hunk 5 above)
Comment (MuthuVega) [reply protocol/0013-ACCT-accounts.md:127]: Yes

File: protocol/0013-ACCT-accounts.md (hunk 6 above)
Comment (MuthuVega) [reply protocol/0013-ACCT-accounts.md:136]: Was changed last week in one place but not here , have removed the duplicate and amended here

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**